configure.ac

```diff
@@ -107,7 +108,16 @@ show_section_title "Configure build tools"
 
 # Find compiler, libtools, etc
 #
+
 LT_INIT([disable-static])
+
+# LT_INIT exposes the ability to configure --enable-static
```

Comment:
Please add an error message if both shared and static libraries are requested. Current version won't build both binaries properly.

Reply:
Please elaborate, why wouldn't the current version build both binaries?

Reply:
because shared library will be build with XLIO_STATIC_BUILD define in this case and socket API will be replaced with xlio_*.

Reply:
The shared library with enabled static is different from the one without it.
Do we need an additional flag to turn on/off the XLIO_STATIC_BUILD?

Reply:
OK